auction-server/src/auction.rs

```diff
@@ -1420,6 +1420,10 @@ impl ChainStore for ChainStoreEvm {
         bids: Vec<Self::SimulatedBid>,
         tx_hash: Vec<u8>,
     ) -> Result<Option<Vec<<Self::SimulatedBid as SimulatedBidTrait>::StatusType>>> {
+        if bids.is_empty() {
```

Comment:
this should work, but i don't completely follow why bids would be empty here? you get the bids for a tx_hash of an auction that was submitted, which should have at least one bid that was submitted, so why would this ever be empty? maybe you could add a comment to clarify this

Reply:
One possible scenario:
Lets say we have two auctions with same tx hash. We try to conclude the first auction and broadcast the result for it's bids. Then the second auction is going to be concluded. The first line of `conclude_auction` if filtering the bids which are submitted. This list is gonna be empty (because the first conclusion changed the bids state). So we will call the `get_bids_result` with an empty bids list